servicemesh-install/topics/install-prerequisites.adoc

```diff
@@ -0,0 +1,252 @@
+
+.Minimum example istio-installation.yaml
+```
+apiVersion: "istio.openshift.com/v1alpha1"
+kind: "Installation"
+metadata:
+  name: "istio-installation"
+spec:
+  jaeger:
+    elasticsearch_memory: 1Gi
+```
+
+NOTE: The custom resource *must* be called `istio-installation`, that is, the metadata value for `name` must be `istio-installation`.
+
+== Custom Resource Parameters
+
+The following tables list the supported custom resource parameters for {ProductName}.
+
+.General Parameters
+|===
+|Parameter |Values |Description | Default
+
+|deployment_type
+|origin, openshift
+|Specifies whether to use Origin (community) or OpenShift (product) default values for unset fields.
+|origin
+|===
+
+.Istio Parameters
+|===
+|Parameter |Values |Description | Default
+
+|authentication
+|true/false
+|Whether to enable mutual authentication
+|false
+
+|community
+|true/false
+|Whether to modify image names to match community images.
+|false
+
+|prefix
+|Any valid image repo
+|Which prefix to apply to Istio image names used in docker pull.
+|If deployment_type=origin the default is `maistra/`
+
+If deployment_type=openshift the default is `openshift-istio-tech-preview/` 
+
+|version
+|Any valid Docker tag
+|Docker tag to use with Istio images
+|0.1.0
+|===
+
+.Jaeger Parameters
+|===
+|Parameter |Values |Description  |Default
+
+|prefix
+|Any valid image repo
+|Which prefix to apply to Jaeger image names used in docker pull.
+|If deployment_type=origin the default is `maistra/`
+
+If deployment_type=openshift the default is `distributed-tracing-tech-preview/`
+
+|version
+|Any valid Docker tag
+|Which Docker tag to use with Jaeger images
+|If deployment_type=origin then 1.6
+
+If deployment_type=openshift then 1.6.0
+
+|elasticsearch_memory
+|Memory size in megabytes or gigabytes.
+|The amount of memory to allocate to the Elasticsearch installation
+|512M
+|===
+
+.Launcher Parameters
+|===
+|Component |Parameter |Description |Default
+
+|openshift
+|user
+|Should be modified to reflect the OpenShift user that you want to use to run the Fabric8 launcher.  
+|`developer`
+
+|
+|password
+|OpenShift user password for running Fabric8 launcher.
+|`developer`
+
+|github
+|username
+|Should be modified to reflect the GitHub user name for the https://help.github.com/articles/signing-up-for-a-new-github-account/[GitHub account] you want to use to run the Fabric8 launcher.  
+|N/A
+
+|
+|token
+|GitHub https://github.com/settings/tokens[personal access token] you want to use to run the Fabric8 launcher.  
+|N/A
+
+|catalog
+|filter
+|Red Hat booster catalog filter
+|booster.mission.metadata.istio
+
+|
+|branch
+|Red Hat Booster catalog to use for Fabric8
+|v35
+
+|
+|repo
+|GitHub repo to use for Fabric8
+|https://github.com/fabric8-launcher/launcher-booster-catalog.git
+|===
+
+
+== Creating the Operator Namespace
+
+NOTE: The custom resource *must* be deployed into the `istio-operator` namespace.
```

Comment:
If this needs to be a note, please use the right syntax.
We also usually put notes like this under the command. If you put it here, it looks like a note for the section instead of the step.